dnsapi/dns_dnsimple.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env sh
+
+# DNSimple domain api
+#
```

Comment:
Please add your github repository link here.  so that one can report issue if there is any bugs.